tests/js/spec/components/smartSearchBar/index.spec.jsx

```diff
@@ -691,32 +753,6 @@ describe('SmartSearchBar', function () {
       expect(searchBar.state.query).toEqual('event.type:error !title:');
     });
 
-    it('removes wildcard', function () {
```

Comment:
doesn't seem like we offer any kind of wildcard autocompletion in the app so I don't think these tests are necessary.

Reply:
upon further investigation it seems like the autocompletion works only in discover search.. hmmm will investigate more

Reply:
i think this has to do with how we fetch tag values where in some cases we are responsive to `*` but in the case of issue search we don't have wildcard autocompletion for tag values (it still works in search, just doesn't give autocompletion hints)

Reply:
either way I'll remove this test from the searchbar since it is already tested in discover search and seems redundant as this is just replacing a tag value with an autocomplete entry which is tested earlier in the test file